lab-gurpreet/lib/promise.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
I would try to use more semantic naming for your files and variables rather than just, for example, 'promise.js' or 'json'.